stable/ambassador/templates/deployment.yaml

```diff
@@ -159,7 +174,7 @@ spec:
             {{- toYaml .Values.resources | nindent 12 }}
         {{- if .Values.pro.enabled }}
         - name: ambassador-pro
-          image: "{{ .Values.pro.image.repository }}:{{ .Values.pro.image.tag }}"
+          image: "{{ .Values.pro.image.repository }}:amb-sidecar-{{ .Values.pro.image.tag }}"
```

Comment:
This would cause breaking changes if you have a copy of the values file with `tag: amb-sidecar-0.6.0` as the chart would then generate `amb-sidecar-amb-sidecar-0.6.0`

Reply:
Ughh good point.

I was trying to think of a way to make this backwards compatible since we only bumped the major version a month ago.

I guess I will just have to do that

Reply:
I think its fine :) even though I also prefer not bumping majors :D I'm guessing its only for a small subset of users anyway.